pypeman/commands.py

```diff
@@ -131,12 +151,14 @@ def main(debug_asyncio=False, profile=False, cli=False, remote_admin=False):
             nodes=nodes,
             endpoints=endpoints,
             channels=channels,
+            plugin_manager=plugin_manager,
             )
         ctx["cli"] = cli
         ctx.update(namespace)
         cli = CLI(namespace=namespace)
         cli.run_as_thread()
 
+    # TODO: might transform remote_admin into a plugin
```

Comment:
Can you fill a ticket instead of this todo ?

Reply:
will do

Reply:
done